build.gradle

```diff
@@ -115,7 +100,6 @@ android {
 
     dexOptions {
         preDexLibraries = true
-        incremental true
```

Comment:
This will slow down builds, but I was getting multiple dex errors during building when it was enabled